src/urllib3/contrib/_securetransport/bindings.py

```diff
@@ -562,26 +564,22 @@ class SecurityConst(object):
     TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 = 0xC030
     TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256 = 0xC02B
     TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 = 0xC02F
-    TLS_DHE_DSS_WITH_AES_256_GCM_SHA384 = 0x00A3
+    TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256 = 0xCCA9
```

Comment:
How are these updated? (How does someone validate that these constants were pulled in correctly?)

Reply:
I saw this comment and I'll address the rest of your comments later but since this chunk is a lot of work to review: https://developer.apple.com/documentation/security/1550981-ssl_cipher_suite_values?language=objc

Reply:
Confirmed. A comment on the source of the constants would be handy. :)

Reply:
Agreed, I'll add this!